lib/compat/wordpress-6.5/fonts/fonts.php

```diff
@@ -164,6 +164,7 @@ function gutenberg_register_font_collections() {
 		array(
 			'name'          => _x( 'Google Fonts', 'font collection name', 'gutenberg' ),
 			'description'   => __( 'Install from Google Fonts. Fonts are copied to and served from your site.', 'gutenberg' ),
+			'permission'    => __( 'To install fonts from Google you must give permission to connect directly to Google servers. The fonts you install will be downloaded from Google and stored on your site. Your site will then use these locally-hosted fonts. You can alternatively upload files directly on the Upload tab.', 'gutenberg' ),
```

Comment:
The `gutenberg_register_font_collections()` function performs an early return if the same Google font collection is already registered in the core. And I don't think this `permission` field exists in the core yet. As a result, I think this `permission` field is not included in the API response. Is my understanding correct?

If so, instead of performing an early reversion, wouldn't it be necessary to unregister the font collection using the `wp_unregister_font_collection()` function and then always re-register the font collection on the Gutenberg plugin side?